ansible/configs/sap-integration/default_vars.yml

```diff
@@ -100,15 +18,19 @@ project_tag: "{{ env_type }}-{{ guid }}"
 
 # Why is this config being deployed? Override for specific environments
 # Some valid: development, ilt, production, event
-purpose: development
+purpose: production
```

Comment:
why change default purpose ?  development seems good to me as a default, and is then overridden in agnosticV